search/search_repository.go

```diff
@@ -265,9 +274,15 @@ func (r *GormSearchRepository) search(ctx context.Context, sqlSearchQueryParamet
 		}
 		db = db.Limit(*limit)
 	}
+	if len(types) > 0 {
+		db = db.Joins("JOIN work_item_types w1 on w1.name=w.type")
```

Comment:
Can we somehow manage to not use the table name hard-coded here? and the line below?